src/solution/my_edge_app/influxdb/init_influxdb.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
+set -e
+
+
```

Comment:
Add comment that in this file we would load secrets from various Secrement management sources and use them to confugre the influx db database.